src/workbench/SideBar.tsx

```diff
@@ -161,23 +161,21 @@ const SideBarContents = ({ panes, ...props }: SideBarContentsProps) => {
         index={index}
       >
         <TabList>
-          <Box
-            width="3.75rem"
-            mt="1.2rem"
-            ml="auto"
-            mr="auto"
-            mb="max(11.5vh, 7.7rem)"
-          ></Box>
+          <Box width="3.75rem" flex={1} maxHeight="8.9rem" minHeight={7}></Box>
           {panes.map((p, i) => (
             <Tab
               key={p.id}
-              color="white"
+              color={p.id === "files" ? "gray.50" : "#F5F6F8"}
               height={width}
               width={width}
               p={0}
               position="relative"
               className="sidebar-tab" // Used for custom outline below
               onClick={handleTabClick}
+              borderRadius={`${cornerSize}px 0 0 ${cornerSize}px`} //temp override - apply to branding
+              pr="6px" //temp override - apply to branding
+              mb={p.id === "reference" ? "auto" : 3}
+              mt={p.id === "files" ? 3 : 0}
```

Comment:
Not a fan of how this switches on id. I'd rather this information was at the level of the parent rather than a child switching on its identity.

I think we should either move this information to `p` (e.g. add a color field etc) or rejig so we're creating a component instance for each tab that takes the information as props and remove the `panes` approach entirely.